### ISSUE TITLE
AlertDialog size and size of corresponding window do no match

### ISSUE THREAD

**Issue by tkuenneth** [JetBrains/compose-multiplatform#282]
The following AlertDialog is too small regarding its window size (spot the white area at the bottom of the window).

```kotlin
AlertDialog(onDismissRequest = {},
    title = { Text("Hello title") },
    text = { Text("Hello text") },
    backgroundColor = Color.LightGray,
    confirmButton = {
      Button(onClick = {}) {
        Text("OK")
      }
    })
```

![image](https://user-images.githubusercontent.com/13284437/104813699-2430e200-580b-11eb-90d5-7831f0d40ac4.png)

The reason is that (deep inside Compose for Desktop) the corresponding window is created with a fixed default size of 400 x 250: 

![image](https://user-images.githubusercontent.com/13284437/104813999-ce5d3980-580c-11eb-9e8c-53e3c4e3f1a0.png)

Now, I could add a modifier to AlertDialog like this:

```kotlin
modifier = Modifier.fillMaxHeight(),
```

![image](https://user-images.githubusercontent.com/13284437/104814075-4166b000-580d-11eb-90f7-9600d2badcba.png)

Then the AlertDialog takes all vertical space, but the button is oddly positioned. So the fix must be applied somewhere here, where the `Column()` is populated:

![image](https://user-images.githubusercontent.com/13284437/104814188-cfdb3180-580d-11eb-811e-313dd9b8c99e.png)

**Comment by m-sasha** [comment]
@tkuenneth Could you clarify what the problem is? With compose 1.3 the alert dialog is vertically centered, as it should be:

<img width="401" alt="image" src="https://user-images.githubusercontent.com/5230206/223364789-59b979e9-5955-4a00-b87c-62f5bf9d7ab6.png">

There does seem to be a different bug, in that it's too narrow, but that doesn't seem to be what you're reporting.

**Comment by m-sasha** [comment]
The issue doesn't appear to be reproducible in the latest Compose, and reporting user isn't responsive.

**Comment by tkuenneth** [comment]
The version on which I reported the bug used **separate** windows for the `AlertDialog()` (as clearly noticeable in my screenshot), and the size of that window simply was too big. Now the alert is no longer shown in a separate window, so obviously the bug no longer surfaces. Therefore closing the issue is perfectly ok. Having said that, waiting two years (I opened the issue Jan 2021) and then complaining about *user isn't responsive' after two weeks is wierd.
